docker-compose.yml

```diff
@@ -2,7 +2,8 @@ version: '3'
 services:
   clickhouse:
       # https://hub.docker.com/r/yandex/clickhouse-server/
-      image: yandex/clickhouse-server:19.13.1.11
+      image: yandex/clickhouse-server:20.1.2.4
```

Comment:
Would be good to start running the test on both the 19.* and 20.* versions of clickhouse for max compat

Reply:
Done